src/scripts/countdown.js

```diff
@@ -0,0 +1,12 @@
+const getTimeFormate = require('./timeFormate');
```

Comment:
Maybe we should call this (and its file) `getFormattedTime` or something similar.